packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -595,6 +598,48 @@ export const RemoteServerFactory = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupSlasPrivateClientProxy(app, options) {
+        if (options.useSlasPrivateClient) {
+            const clientSecret = process?.env?.SLAS_PRIVATE_CLIENT_SECRET
+            if (!clientSecret) {
+                app.use('/ssr/auth', (_, res) => {
```

Comment:
I think ideally we'd choose a path name that makes it clear that this is a managed endpoint and also that its a proxy for SLAS. Perhaps something like `/mobify/scapi/shopper/auth/`.